opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -178,12 +185,20 @@ def get_empty() -> "Resource":
     def attributes(self) -> Attributes:
         return self._attributes.copy()
 
+    @property
+    def schema_url(self) -> Attributes:
```

Comment:
Shouldn't this return either a string (or I don't know if a Schema_url object would be a good idea)?
```suggestion
    def schema_url(self) -> str:
```

Reply:
Done! Silly typo. I personally don't see the need for a type.